pkg/app/server/githubstore/store.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package githubstore
```

Comment:
I wonder about the need for this package.
Currently, it is quite simple so which way is better: adding this package or implementing it directly in the API. 🤔

Reply:
I thought the same too, but looks like the current web_api.go does not import any 3rd packages so I think maybe this package is needed 😅

Reply:
There was no special reason for that fact. Just because we do not need any 3rd packages yet.
So importing that `github` package directly is fine to me.

Reply:
okay, lets me address that. Thanks for clarifying 🙏